src/test/kotlin/lotto/domain/LottoTicketTest.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+internal class LottoTicketTest {
+
+    @DisplayName(value = "6개가 아닌 로또 번호를 입력했을 때 예외 발생")
+    @Test
+    fun checkWinning() {
+        Assertions.assertThatThrownBy {
+            LottoTicket(setOf(LottoNumber(10), LottoNumber(20)))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
            LottoTicket(10, 20)
```